### PR TITLE
fix thumbnail reset when new default style is set

### LINKED ISSUE #426: force thumbnail reset when new default style set

### PR DESCRIPTION
**Pull request by boney-bun**
fix #426

### CODE REVIEW

**Comment by boney-bun** [comment]
the PR in action:
changedefaultstyle

**Comment by gubuntu** [comment]
fix so you can merge asap @boney-bun

File: geonode/qgis_server/views.py [hunk 1]
```diff
@@ -838,9 +838,14 @@ def default_qml_style(request, layername, style_name=None):
         # Give thumbnail creation to celery tasks, and exit.
         bbox_string = layer.bbox_string
         bbox = bbox_string.split(',')
+        bbox_srid = layer.bbox[-1]
```
Comment (lucernae) [inline geonode/qgis_server/views.py]:
```suggestion
        bbox_srid = layer.srid
```

File: geonode/qgis_server/tests/test_views.py [hunk 2]
```diff
@@ -564,30 +564,29 @@ def test_add_delete_style(self):
...
+        # Check new default, namely: new_style
         default_style_url = reverse(
             'qgis_server:default-qml',
             kwargs={
-                'layername': layer.name})
+                'layername': layer.name,
+                'style_name': 'new_style'})
 
-        response = self.client.get(default_style_url)
+        response = self.client.post(default_style_url)
```
Comment (lucernae) [inline geonode/qgis_server/tests/test_views.py]: Why you post? The test below is to test default style query (using get). Maybe you want to create another section tests?

Comment (boney-bun) [reply geonode/qgis_server/tests/test_views.py]: yes, i have created a new test section for the post request. 
the post sets a new style.

**Comment by boney-bun** [comment]
thanks for the feedback @lucernae 
this PR intends to fix a bug when setting a new default style.
the PR handles the non 4326 SRS.

i'm going to merge this PR and bring it to testing.

**Pull request merged**